scripts/jenkins-yoctobuild-build.sh

```diff
@@ -689,26 +591,15 @@ build_and_test_client() {
             # run tests with xdist explicitly disabled
             local qemu_testing_status=0
             if $python3_supported; then
-                python3 -m pytest -p no:xdist --verbose --junit-xml=results.xml $host_args \
+                python3 -m pytest -p no:xdist --verbose --junit-xml=results.xml \
                         --bitbake-image $image_name --board-type=$board_name $pytest_args \
                         $html_report_args $acceptance_test_to_run || qemu_testing_status=$?
             else
-                py.test -p no:xdist --verbose --junit-xml=results.xml $host_args \
+                py.test -p no:xdist --verbose --junit-xml=results.xml \
                     --bitbake-image $image_name --board-type=$board_name $pytest_args \
                     $html_report_args $acceptance_test_to_run || qemu_testing_status=$?
             fi
 
-            if [ $qemu_testing_status -ne 0 ]; then
-                if is_hardware_board "$board_name"; then
-                    board_support_update "$board_name" "failed"
-                fi
-                exit $qemu_testing_status
-            else
-                if is_hardware_board "$board_name"; then
-                    board_support_update "$board_name" "passed"
-                fi
-            fi
-
             if [ $qemu_testing_status -ne 0 ]; then
                 exit $qemu_testing_status
             fi
```

Comment:
Same here, can be removed along with `qemu_testing_status` assignments.

Reply:
:+1: